mp_api/client/mprester.py

```diff
@@ -1477,3 +1477,148 @@ def query(*args, **kwargs):
             please see the new documentation.
             """
         )
+
+    def get_cohesive_energy(
+        self,
+        material_id: MPID | str | list[MPID | str],
```

Comment:
I think it would be better practice / more consistent to use `material_ids` here and only accept a list of mp-ids (the list could be len==1). That would also make the return value consistent and always a dict mapping mp-id to float.

Reply:
Sounds good to me - changed